src/wesnoth.cpp

```diff
@@ -1107,7 +1107,7 @@ int main(int argc, char** argv)
 			PLAIN_LOG << "Started on " << ctime(&t);
 		}
 
-		const std::string& exe_dir = filesystem::get_exe_dir();
+		const std::string exe_dir = filesystem::get_exe_dir();
```

Comment:
This line was rewritten in #7145, so it's a merge conflict to change it here.

Please rebase your changes, and force-push the new version of the branch.

Reply:
dang, missed it

Reply:
if you get to see my very weird rebase... how I managed to rebase like that, not sure, fixing it now

Reply:
done